Log/Message/Entry.ts

```diff
@@ -24,4 +24,15 @@ export namespace Entry {
 			  }
 			: undefined
 	}
+	export function getLocationEntry(request: any): Entry {
+		return {
+			message: "Locations",
+			data: {
+				"cf-connecting-ip": request.headers["cf-connecting-ip"],
+				"cf-ipcountry": request.headers["cf-ipcountry"],
+				datacenter: request.cf.colo,
+				country: request.cf.colo,
+			},
```

Comment:
Why say the request.cf.colo twice? Remove one.

Reply:
fixed